FWCore/Framework/interface/EDLooperBase.h

```diff
@@ -27,25 +27,23 @@
    use an EventID to exactly specify what next event you want.  However, if you say to go 'back' one event while the
    job is on the first event of the loop or pass an EventID for an event not contained in the source the job will
    immediately go to 'end of loop'.
-   NOTE: if you have no need of controlling exactly what 'next' event should be processed then you should instead inherit
-   from the subclass EDLooper and us it simplified interface for 'duringLoop'.
- 3) end of loop
-   Once all events have been processed or kStop was returned from 'duringLoop' or ProcessingController was told to go to an
-   'invalid' event then the method
-      Status endOfLoop(EventSetup const&, unsigned int iCounter)
-   will be called.  iCounter will be the number of loops which have been run in the job starting at index 0. If kContinue
-   is returned from endOfLoop then a new loop will begin else if kStop is called then the job will end.
-
- Like other modules, an EDLooperBase is called for 'beginJob', 'endJob', 'beginRun', 'endRun', 'beginLuminosityBlock' and
- 'endLuminosityBlock'.
+   NOTE: if you have no need of controlling exactly what 'next' event should be processed then you should instead
+ inherit from the subclass EDLooper and us it simplified interface for 'duringLoop'. 3) end of loop Once all events have
```

Comment:
It changed the formatting of the comment. It did not preserve the alignment of the paragraph.

Reply:
I can try to turn off comment reformatting (seems this is possible)